backend/src/apiserver/server/pipeline_server.go

```diff
@@ -224,7 +224,7 @@ func (s *PipelineServer) CreatePipelineVersion(ctx context.Context, request *api
 		return nil, util.Wrap(err, "The URL is valid but pipeline system failed to read the file.")
 	}
 
-	version, err := s.resourceManager.CreatePipelineVersion(request.Version, pipelineFile)
+	version, err := s.resourceManager.CreatePipelineVersion(request.Version, pipelineFile, true)
```

Comment:
nit: can you update CreatePipelineVersion endpoint to also allow this new option?
You'll need to edit this proto: https://github.com/kubeflow/pipelines/blob/fffe830ba9b1ce0b1bb1ae910fcbb9b572e11baa/backend/api/pipeline.proto#L100.

and regenerate server/client boilerplate code by running:
1. https://github.com/kubeflow/pipelines/tree/master/backend#building-go-client-library-and-swagger-files
2. https://github.com/kubeflow/pipelines/blob/master/backend/api/build_kfp_server_api_python_package.sh

Reply:
Should be fixed now.

Reply:
Hmmm, did you forget to pass the parameter here?

Reply:
Ohhh , yes I did, I actually misunderstood some of the logics completely.